### DISCUSSION TITLE
Is there a way to divide the Tenant?(Group Tenant)

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [filamentphp/filament#11846] [Help]
### Package

Panel builder

### Package Version

3.2.47

### How can we help you?

Is there a way to divide the Tenant? (Group Tenant)

Example:
There are multiple companies, and each company has at least one branch:

Company 1
- Branch 1
- Branch 2

Company 2
- Branch 1
- Branch 2

Can the drop-down menu be divided like this?

**Comment by micraux** [comment]
Hello,

I don't think you can have a "tenant belongs to tenant" structure.

Depending on what you want to achieve, you could use the tenant feature for companies, and create a `branch` model belonging to `company` model.

Then, you could use roles and permissions to allow user access in a `company` and/or `branch`.
